tests/multicomponent_steinberger.prm

```diff
@@ -0,0 +1,131 @@
+## Test the multicomponent steinberger material model. Three blobs with different viscosities move around. The reference viscosity of background is from the viscosity profile of Steinberger & Calderwood 2006.
+
+set Dimension = 2
+set CFL number                             = 1.0
+set End time                               = 60000
+set Resume computation                     = false
+set Start time                             = 0
+set Adiabatic surface temperature          = 1600
+set Surface pressure                       = 0
+set Pressure normalization = surface
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, single Stokes
+set Output directory = /home/poulami/software/Aspect_globalscale/aspect/tests/multicomponent_steinberger
```

Comment:
Can you remove that line? It still refers to the location on your computer. 
(And for a test, this will be chosen automatically anyway).